scripts/upgrade_admin_server.sh

```diff
@@ -119,6 +119,10 @@ ListenStream=[::]:4369
 FreeBind=true
 EOF
 
+    # Make sure to do schema migration properly after packages were upgraded
+    pushd /opt/dell/crowbar_framework
+    sudo -u crowbar RAILS_ENV=production bin/rake crowbar:schema_migrate_prod
```

Comment:
perhaps we should use an absolute path for bin/rake here?

Reply:
Hm, good point. I've used this rake as it has been used from this script before (see https://github.com/crowbar/crowbar/pull/2347) and indeed there's `/opt/dell/crowbar_framework/bin/rake` which we package with crowbar-core package... 
It probably does not make a difference in this call though

Reply:
Fair enough, but we should popd back :-)

Reply:
done